src/replicator.rs

```diff
@@ -156,7 +155,7 @@ impl Replicator {
             Self::poll_for_last_id_and_entry_height(&cluster_info)?;
 
         let signature = keypair.sign(storage_last_id.as_ref());
-        let (entry_height, max_entry_height) =
+        let (entry_height, _max_entry_height) =
             get_entry_heights_from_last_id(&signature, storage_entry_height);
```

Comment:
Looks like this is the only use of this function. Why not change it to return just the one value?

Reply:
Wasn't sure if that max_entry_height would still have some utility in the future. As Steven has acknowledged it doesn't, I'll remove it now :)

Reply:
Delete anything and everything that's not in use. It's all in git. We know where to find it if we want it back.